packages/server/api/model/supportReference.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
```

Comment:
Change this file name to `referenceModel` to follow the naming pattern for models.

Reply:
Also, claimModel and claimReviewModel should now have a field for a relationship with `reference` table. One claim or claimReview can have many references.